docs/quickstart.md

```diff
@@ -1,60 +1,188 @@
+
+## Prerequisites
+
+- [Golang](https://golang.org/dl/) version 1.13 or above. You can follow the instructions in the official [installation page](https://golang.org/doc/install) (check it by `go version`)
+
+## Create a Project
+
+```bash
+$ mkdir colx && cd colx
+$ go mod init colx && touch main.go
+```
+
+## Import Dependencies
+
+First of all, you need to use `go get` to fetch the dependencies:
+
+```bash
+$ go get -v github.com/pingcap/parser@v4.0.0-rc.1
+```
+
+> **NOTE**
+>
+> If you want to use advanced API on expression(which is a kind of AST node), such as numbers, string literals, booleans, nulls, etc., it is strongly recommended to use TiDB's `types` package with the following command:
+>
+> ```bash
+> go get -v github.com/pingcap/tidb/types/parser_driver@v4.0.0-rc.1
+> ```
+> and import it in the directive of your golang source code:
+> ```go
+> import _ "github.com/pingcap/tidb/types/parser_driver"
+> ```
+
+Your directory should contains three files:
+```
+.
+├── go.mod
+├── go.sum
+└── main.go
+```
+
+Now, open `main.go` with your favouriate editor, start coding!
+
+## Parse SQL text
+
+To convert a SQL text to an AST tree, you can:
+1. use `parser.New()` to instantiate a parser and
+2. call the method `Parse(sql, charset, collation)` on it.
 
 ```go
+package main
+
 import (
 	"fmt"
 	"github.com/pingcap/parser"
-	_ "github.com/pingcap/tidb/types/parser_driver"
+	"github.com/pingcap/parser/ast"
+	_ "github.com/pingcap/parser/test_driver"
 )
 
-func example() {
+func parse(sql string) (*ast.StmtNode, error) {
 	p := parser.New()
 
-	stmtNodes, _, err := p.Parse("select * from tbl where id = 1", "", "")
+	stmtNodes, _, err := p.Parse(sql, "", "")
+	if err != nil {
+		return nil, err
+	}
 
-	fmt.Println(stmtNodes[0], err)
+	return &stmtNodes[0], nil
+}
+
+func main() {
+	astNode, err := parse("SELECT a, b FROM t")
+	if err != nil {
+		fmt.Printf("parse error: %v\n", err.Error())
+		return
+	}
+	fmt.Printf("%v\n", *astNode)
 }
 ```
 
-Let's explain the details line by line.
+Test it with
 
-## Import driver
+```bash
+$ go run main.go
+```
 
-```go
-import _ "github.com/pingcap/tidb/types/parser_driver"
+You should get something like this:
+
+```
+&{{{{SELECT a, b FROM t}}} {[]} 0xc0000a1980 false 0xc00000e7a0 <nil> 0xc0000a19b0 <nil> <nil> [] <nil> <nil> none [] false false 0 <nil>}
 ```
 
-This means we are using the parser driver provided by TiDB, which decides how to parse the basic data types in SQL, such as numbers, string literals, booleans, nulls, etc.
+> **NOTE**
+>
+> Here are a few things you might want to know:
+> - To use a parser, a `parser_driver` is required. It decides how to parse the basic data types in SQL.
+>
+>   You can use `github.com/pingcap/parser/test_driver` as the `parser_driver` for test. Again, if you need advanced features, please use the `test_driver` in TiDB(`go get -v github.com/pingcap/tidb/types/parser_driver@v4.0.0-rc.1` and import it).
```

Comment:
```suggestion
>   You can use `github.com/pingcap/parser/test_driver` as the `parser_driver` for test. Again, if you need advanced features, please use the `test_driver` in TiDB (`go get -v github.com/pingcap/tidb/types/parser_driver@v4.0.0-rc.1` and import it).
```

BTW why 4.0.0-rc.1 ?

Reply:
I found the newest [released parser version](https://github.com/pingcap/parser/releases) is 4.0.0-rc.1.